moncli/entities/column_value.py

```diff
@@ -534,64 +534,53 @@ def __init__(self, **kwargs):
     def lat(self):
         """ The latitude value"""
         try:
-             return loads(self.value)['lat']
+            return loads(self.value)['lat']
         except KeyError:
-            return None
+            return self.value
         
     @lat.setter
     def lat(self, latitude):
-        print("lat setter")
-        latitude = self.lat
-        if (float(latitude) > 90) or (float(latitude) > -90) :
+        if latitude >= -90 and latitude <= 90:
+            print("latitude = ",latitude)
             return self.set_value(lat=latitude)
         else:
             raise LocationError(latitude)
+        
    
     @property
     def lng(self):
         """ The longitude value"""
         try:
-             return loads(self.value)['lng']
+            return loads(self.value)['lng']
         except KeyError:
-            return None
-        
+            return self.value
+
+
     @lng.setter
     def lng(self, longitude):
-        print("lng setter")
```

Comment:
None values still need to be accounted for.  We can do this by adding the following code before the longitude value check on line 561:

`
try:
    float(longitude)
except TypeError:
    if not longitude: # This is our null check
        return self.set_value(lng=longitude)
    raise LocationError("Invalid Longitude")
`